### PR TITLE
Update use case pages to use shortcodes

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/use-cases/configure.md [hunk 1]
```diff
@@ -11,54 +11,69 @@ You can get a smart machine running with Viam in just a few steps.
...
-</table>
+{{< table >}}
+{{< tablestep >}}
+
+{{<imgproc src="/use-cases/signup-narrow.png" class="fill alignleft" resize="500x" style="max-width: 200px" declaredimensions=true alt="Viam app login screen.">}}
+**1. Create a machine in the Viam app**
+
+First, [create a Viam account](https://app.viam.com/) if you haven't already. Log in.
+
+Then create a machine by typing in a name and clicking **Add machine**</strong>.
```
Comment (andf-viam) [inline docs/use-cases/configure.md:22]:
```suggestion
Then create a machine by typing in a name and clicking **Add machine**.
```

I think this is what's causing Item 4 to render asterisks in the staged view.

File: docs/use-cases/image-data.md [hunk 2]
```diff
@@ -11,46 +11,54 @@ Once you have synced your images, you can view them in the Viam app, filter your
...
+Then [add a camera component](/components/camera/), such as a [webcam](/components/camera/webcam/).
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/services/icons/data-management.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Collect data">}}
+**2. Configure the data management service**
+
+Next, [add the data management service](/data/) to be able to configure how your camera captures and stores images.
+
+Then configure [data capture](/data/capture/)and [cloud sync](/data/cloud-sync/).
```
Comment (andf-viam) [inline docs/use-cases/image-data.md:30]:
```suggestion
Then configure [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/).
```

File: docs/use-cases/image-data.md [hunk 3]
```diff
@@ -11,46 +11,54 @@ Once you have synced your images, you can view them in the Viam app, filter your
...
+**3. Capture data**
+
+With data management configured, [capture image data from a camera on your machine](/data/capture/#configure-data-capture-for-individual-components). Captured data is automatically synced to the cloud after a short delay.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/ml/collect.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
+**4. View data in the Viam app**
+
+Once you have synced images, you can [view those images in the Viam app](/data/view/)from the **Data** tab.
```
Comment (andf-viam) [inline docs/use-cases/image-data.md:44]:
```suggestion
Once you have synced images, you can [view those images in the Viam app](/data/view/) from the **Data** tab.
```

File: docs/use-cases/navigate.md [hunk 4]
```diff
@@ -13,36 +13,42 @@ images: ["/services/icons/navigation.svg"]
...
-    </td>
-  </tr>
-</table>
+{{< table >}}
+{{< tablestep >}}
+{{<imgproc src="/use-cases/base-control.png" class="fill alignright" resize="200x" style="max-width: 200px" declaredimensions=true alt="Base control card">}}
+**1. Teleoperate**
+
+You can remotely control your rover from anywhere through the [Viam app](https://app.viam.com).
+Create an account and add a machine, [install viam-server](/get-started/installation/), and [configure your rover base](/components/base/).
```
Comment (andf-viam) [inline docs/use-cases/navigate.md:22]: Nit, inherited: monospace.

```suggestion
Create an account and add a machine, [install `viam-server`](/get-started/installation/), and [configure your rover base](/components/base/).
```

File: docs/use-cases/manage-fleet.md [hunk 5]
```diff
@@ -17,38 +17,48 @@ For example, you might have 30 robots in one warehouse and 500 in another.
...
+When you [install `viam-server`](/get-started/installation/) on each machine, unique keys are generated to securely connect it to the Viam app.
+Use the config builder interface to configure {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} for new or existing smart machines.
+You can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} to streamline the process of configuring multiple similar machines.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/use-cases/last-online.png" class="fill alignright" resize="600x" style="max-width: 100px" declaredimensions=true alt="Machine last online status indicator in the Viam app.">}}
+**5. Monitor your fleet**
+
+View the status and logs from each machine in the [https://app.viam.com](Viam app), gather machine sensor [data](/data/), and operate machines remotely from the [**Control** tab](/fleet/machines/#control).
```
Comment (andf-viam) [inline docs/use-cases/manage-fleet.md:57]: Nit: what does "gather machine sensor data" mean over "view the status and logs in the Viam app" ?

Maybe something like:

```suggestion
In the [Viam app](https://app.viam.com), you can:
- Monitor the status of each machine from its [**Logs** tab](/fleet/machines/#logs).
- View any data captured by your fleet from the [**Data** tab](/data/).
- Operate machines remotely from the [**Control** tab](/fleet/machines/#control).
```

File: docs/use-cases/manage-fleet.md [hunk 6]
```diff
@@ -17,38 +17,48 @@ For example, you might have 30 robots in one warehouse and 500 in another.
...
-      <p>View the status and logs from each machine in the <a href="https://app.viam.com">Viam app</a>, gather machine sensor <a href="/data/">data</a>, and operate machines remotely from the <a href="/fleet/machines/#control"><strong>Control</strong> tab</a>. Use <a href="/registry/">modules</a> to deploy code to your fleet and manage versioning.</p>
-    </td>
-  </tr>
-</table>
+{{< table >}}
+{{< tablestep >}}
+{{<imgproc src="/use-cases/signup.png" class="fill alignright" resize="600x" style="max-width: 350px" declaredimensions=true alt="Viam app signup screen">}}
+**1. Create an account**
+
+Go to the [https://app.viam.com](Viam app) and sign up with Google, GitHub, Apple, or an email address.
```
Comment (andf-viam) [inline docs/use-cases/manage-fleet.md:25]:
```suggestion
Go to the [Viam app](https://app.viam.com) and sign up with Google, GitHub, Apple, or an email address.
```

**Review by andf-viam — APPROVED** [review]
LGTM % few small fixes. Thanks for making shortcodes out of these semi-standardized HTML layouts, much easier to work with!

File: docs/use-cases/manage-fleet.md [hunk 7]
```diff
@@ -17,38 +17,51 @@ For example, you might have 30 robots in one warehouse and 500 in another.
...
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/use-cases/last-online.png" class="fill alignright" resize="600x" style="max-width: 100px" declaredimensions=true alt="Machine last online status indicator in the Viam app.">}}
+**5. Monitor your fleet**
+
+Using the [Viam app](https://app.viam.com), you can:
+- Monitor the status of each machine from its [**Logs** tab](/fleet/machines/#logs).
+- View any data captured by your fleet from the [**Data** tab](/data/).
+- Operate machines remotely from the [**Control** tab](/fleet/machines/#control).
+Use [modules](/registry/) to deploy code to your fleet and manage versioning.
```
Comment (npentrel) [inline docs/use-cases/manage-fleet.md:61]:
```suggestion

- Monitor the status of each machine from its [**Logs** tab](/fleet/machines/#logs).
- View any data captured by your fleet from the [**Data** tab](/data/).
- Operate machines remotely from the [**Control** tab](/fleet/machines/#control).

Use [modules](/registry/) to deploy code to your fleet and manage versioning.
```

**Pull request merged**